lib/util/template/index.ts

```diff
@@ -1,7 +1,108 @@
+import is from '@sindresorhus/is';
+import safeStringify from 'fast-safe-stringify';
 import * as handlebars from 'handlebars';
+import { logger } from '../../logger';
 
 handlebars.registerHelper('encodeURIComponent', encodeURIComponent);
 
-export function compile(template: string, input: any): string {
+// istanbul ignore next
```

Comment:
```suggestion
```
This gets calleb by `lib/util/template/index.spec.ts`

Reply:
?? Shouldn't we have a test for this? To be safe? I know it's trivial but we should use ignore only for untestable scenarios